### PR TITLE
Views

### PR DESCRIPTION
**Pull request by anderCM**
- Implement the design from the sneak peek wireframes.
  - Styling is nice to have but focus on making sure everything functions first. (Note: Ignore login logic, you would be using Devise in coming milestone)

| | |
|--------|--------|
| blog_root_users | blog_user_page |
| blog_user_all_posts | blog_single_post|

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,11 @@
-<h1>User Details</h1>
\ No newline at end of file
+<div class="container my-3">
+  <%= render partial: 'shared/user', locals: { user: @user } %>
+  <%= render partial: 'shared/bio', locals: { user: @user } %>
+
+  <% @posts.each do |post| %>
```
Comment (Davidosky007) [inline app/views/users/show.html.erb:5]: * Kindly use the `#recent_post` method to render the user's 3 most recent posts as shown in the wireframe 
image

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1 +1,12 @@
-<h1>List of posts here</h1>
\ No newline at end of file
+<div class="container my-3">
+  <%= render partial: 'shared/user', locals: { user: @user } %>
+  <%= render partial: 'shared/bio', locals: { user: @user } %>
+
+  <% @posts.each do |post| %>
```
Comment (Davidosky007) [inline app/views/posts/index.html.erb:5]: * Kindly use the `post.recent_comments` to render 5 most recent comments as shown in the wireframe provided 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @anderCM 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* Views associated with the Models and Controllers were added properly 👍 
* Well written PR 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/show.html.erb (hunk 1 above)
Comment (anderCM) [reply app/views/users/show.html.erb:5]: @Davidosky007  Thanks I noticed that I have already created the method `last_3_posts`. I will reuse it meet the requirements

File: app/views/posts/index.html.erb (hunk 2 above)
Comment (anderCM) [reply app/views/posts/index.html.erb:5]: @Davidosky007  Thanks I noticed that I have already created the method `last_5_comments`. I will reuse it meet the requirements

**Review by Owusu-Desmond — APPROVED** [review]
Hi @anderCM , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**